avocado/core/job.py

```diff
@@ -371,31 +370,15 @@ def _log_avocado_datadir():
         job_log.info('logs     ' + data_dir.get_logs_dir())
         job_log.info('')
 
-    def _log_variants_tree(self, variant):
```

Comment:
That's my point.  In this specific commit, we loose it.  I doubt someone will checkout this commit and complain about it, but this indicates that the approach I suggested ealier (bring the new `str*()` utilities first, then replace the old functions with it) causes less (ideally no) disruption.